.gitmessage

```diff
@@ -0,0 +1,37 @@
+
+# [<tag>] <subject> (Max 80 char)
+# |<----   Preferably using up to 60 chars   --->|<------------------->|
+# Example: [feature] Implement automated commit messages
+
+# (Optional) Explain why this change is being made
+# |<----   Try To Limit Each Line to a Maximum Of 80 Characters   ---->|
+
+# --- COMMIT END ---
+# Tag can be
+#    analyzer      (Analyzer related changes)
+#    cmd           (Command line changes)
+#    cfg           (Config file changes)
+#    db            (Issues related to the database schema)
+#    doc           (Changes to documentation)
+#    gui           (Changes in the web GUI)
+#    fix           (Bug fix)
+#    feature       (New feature)
+#    plist2html    (Changes in the plist-to-html tool)
+#    refactor      (Refactoring code)
+#    server        (Server related changes)
+#    style         (Formatting, missing semi colons, etc; no code change)
+#    test          (Adding or refactoring tests; no production code change)
+#    tu-collector  (Changes in the tu-collector tool)
+#    version       (Version bump/new release; no production code change)
+#    web           (Changes in the web module)
+#
+# Note: Multiple tags can be combined, e.g. [fix][gui] Fix issue X with methodhandles
```

Comment:
would it be a bit easier to read and write multiple tags with this format: `[fix, gui]`  
`[db][gui][server][api][fix]`
`[db, gui, server, api, fix]`
`[db,gui,server,api,fix]` more dense to save some characters


what is the `methodhandles`?

Reply:
I don't think that it would be easier to read. You are right that we would save some characters with this but it would be worse to find for example gui related commits:
```sh
# git log --pretty="%h - %s" | grep "gui"
3ab54289 - user guide fix
52d5b5f8 - User guide for tidy version
f4d39880 - Update user_guide.md
4ec33d91 - Update user_guide.md
15ba961d - update user guide, keep temporary reports
3d1b60f8 - Update user_guide.md
af190d39 - Update user_guide.md
```
If we separate the tags with brackets, it is much better to find tag specific commits.
```sh
# git log --pretty="%h - %s" | grep "\[gui\]"
ae0067fe - [gui] Fix delete run
4aca70de - [gui] Handle thrift api call exceptions
```